app/controllers/admin/items_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
   def destroy
     item = Item.find(params[:id])
     item.destroy
-    flash[:deleted] = 'You have removed the item from the store.'
+    flash[:deleted] = 'You have removed an item from the store, brah.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.